### ISSUE TITLE
CQ500 data exploration

### ISSUE THREAD

**Issue by niladell** [niladell/unsupervised-medical-learning#6]
We need to do a bit of data exploration of the CQ500, which basically means building the baselines we mentioned, based on what has classically been done in disentanglement studies for the other datasets. Perform PCA analysis, and then check if/how the projected data clusters (everything fairly simple with sk-learn); also give a try on how t-SNE looks like.

To do all of this, I'd create a new branch specially for that, and work in an independent folder (name it experiments or something like that if you want). About the tool, maybe the best is to use Jupiter notebooks. Before anything else, tho, check #1 about how the data is encoded and how to load it. Also if during this task, something is missing or needs to be changed about #1, mention it, so we can polish the way we create the datafiles.

**Comment by InezPereira** [comment]
Agreed. Before that, I am looking at the existing subfolders to be sure we have homogeneous data and don't mix up different window types and contrast with no contrast.

**Comment by pjaweh** [comment]
Another issue: How many of the folders actually contain only 1 dcm image?
